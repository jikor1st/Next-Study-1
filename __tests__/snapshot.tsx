import { render } from '@testing-library/react';
import Testing from '@/pages/testing';

it('renders Testing unchanged', () => {
  const { container } = render(<Testing />);
  expect(container).toMatchSnapshot();
});
