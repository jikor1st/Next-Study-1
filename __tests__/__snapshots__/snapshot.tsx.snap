// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Testing unchanged 1`] = `
<div>
  <div>
    <p>
      테스팅 페이지
    </p>
  </div>
</div>
`;
